backend/canisters/community/api/src/queries/api_key.rs

```diff
@@ -13,8 +13,21 @@ pub struct Args {
 #[allow(clippy::large_enum_variant)]
 #[derive(Serialize, Deserialize, Debug)]
 pub enum Response {
-    Success(ApiKey),
+    Success(String),
     NotAuthorized,
     NotFound,
     ChannelNotFound,
+    CommunityFrozen,
+}
+
+impl From<Response> for types::c2c_bot_api_key::Response {
+    fn from(value: Response) -> Self {
+        match value {
+            Response::Success(s) => types::c2c_bot_api_key::Response::Success(s),
+            Response::CommunityFrozen => types::c2c_bot_api_key::Response::Frozen,
```

Comment:
We might as well remove `CommunityFrozen`, it doesn't matter if the community is frozen